### PR TITLE
Remove the arch parameter from option

### PR DESCRIPTION
**Pull request by xiangge**
Remove the arch parameter from option.

JIRA: PDC-2076

### CODE REVIEW

File: pdc_client/plugins/release_variant.py [hunk 1]
```diff
@@ -52,14 +52,17 @@ def register(self):
...
+    def add_release_variant_arguments(self, parser, required=False, exclude=False):
         required_args = OrderedDict()
         required_args["release"] = {}
         required_args["uid"] = {}
         required_args["id"] = {}
         required_args["name"] = {}
         required_args["type"] = {}
         required_args["arch"] = {"action": "append", "dest": "arches"}
+        if exclude:
+            required_args.pop("arch")
```
Comment (hluk) [inline pdc_client/plugins/release_variant.py]: Argument `exclude` is not very clear. Perhaps:

```python
if requires_arg_arch:
    required_args["arch"] = {"action": "append", "dest": "arches"}
```

**Review by hluk — CHANGES_REQUESTED** [review]
paramter -> parameter

File: pdc_client/plugins/release_variant.py (hunk 1 above)
Comment (xiangge) [reply pdc_client/plugins/release_variant.py]: I was thinking about maybe some parameters also need to pop in the future, so just use the exclude here.
Anyway, use exclude_arches to instead now.

**Review by hluk — APPROVED** [review]

**Comment by simozhan** [comment]
LGTM

**Pull request merged**